packages/openapi-code-generator/src/typescript/common/schema-builders/zod-schema-builder.spec.ts

```diff
@@ -546,6 +546,20 @@ describe.each(testVersions)(
 
         await expect(execute(undefined)).resolves.toBe(0)
       })
+
+      it("supports default values of null when nullable", async () => {
+        const {code, execute} = await getActualFromModel({
+          ...base,
+          nullable: true,
+          default: null,
+        })
+
+        expect(code).toMatchInlineSnapshot(
+          '"const x = z.coerce.number().nullable().default(null)"',
+        )
+
+        await expect(execute(undefined)).resolves.toBeNull
```

Comment:
```suggestion
        await expect(execute(undefined)).resolves.toBeNull()
```

(same for all the `toBeNull` assertions)

Reply:
[fix](https://github.com/mnahkies/openapi-code-generator/pull/360/commits/5d4baa23fe8388e707046b6541462df80a036386)